ramenctl/ramenctl/unconfig.py

```diff
@@ -39,10 +39,24 @@ def run(args):
     # We keep the ramen config map since we do not own it.
 
     command.info("Deleting s3 secret in ramen hub namespace")
+    template = drenv.template(command.resource("ramen-s3-secret.yaml"))
     kubectl.delete(
-        "--filename",
-        command.resource("ramen-s3-secret.yaml"),
+        "--filename=-",
         "--ignore-not-found",
+        input=template.substitute(namespace=args.ramen_namespace),
         context=env["hub"],
         log=command.debug,
     )
+
+    template = drenv.template(command.resource("cloud-credentials-secret.yaml"))
+    yaml = template.substitute(cloud="", namespace=args.ramen_namespace)
+
+    for cluster in env["clusters"]:
+        command.info("Deleting cloud credentials secret in clsuter '%s'", cluster)
```

Comment:
I don't think the typo is fixed. Did you push the update?

Reply:
Oh, this is in a different file. Please update this file too. Rest of the changes look good.

Reply:
Fixing, thanks!